homeassistant/components/aemet/__init__.py

```diff
@@ -37,6 +39,8 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     except TownNotFound as err:
         _LOGGER.error(err)
         return False
+    except asyncio.TimeoutError as err:
```

Comment:
You should probably catch this internally in the library to avoid leaking details that its using aiohttp behind the scenes. But good for now.

Reply:
Ok, I'll do that, thanks!